index.bs

```diff
@@ -753,6 +753,62 @@ host it MUST perform the following steps for each script execution environment:
   };
 </pre>
 
+<div class="example">
+In this example we imagine a data logging device supporting up to 8 16-bit data
+channels. When data is sampled it is available on endpoint 1, which is a bulk IN
+endpoint. The maximum packet size on this endpoint is 16 bytes to support all 8
+channels being activated at the same time. To save bus bandwidth, however, any
+combination of channels can be activated and deactivated. The packet will only
+be the length necessary to transmit the data collected.
+
+To get started we open the device, select the first configuration (in case the
+operating system has not already done this during enumeration) and claim the
+data logging interface,
+
+<pre highlight="js">
+  await device.open();
+  if (device.configuration === null)
+    await device.setConfiguration(1);
+  await device.claimInterface(1);
+</pre>
+
+For this particular application we care about reading from channels 1, 2 and 5
+so we issue a control transfer to activate these channels,
+
+<pre highlight="js">
+  await device.controlTransferOut({
+      requestType: 'vendor',
+      recipient: 'interface',
```

Comment:
Would it make sense to link these constants to their definitions elsewhere in the spec?